axiom/datasets.go

```diff
@@ -499,6 +507,10 @@ func (s *DatasetsService) IngestEvents(ctx context.Context, id string, events []
 // The method returns without an error if the channel is closed and the buffered
 // events are successfully sent to the server.
 //
+// The returned ingest status does not contain a trace ID as the underlying
```

Comment:
I guess it's not worth aggregating all the trace IDs and exposing a slice?

Reply:
I thought about that, as well. It feels a bit.. odd? Don't you think? I guess people might want to use the TraceID in their logging or own instrumenting. From a user perspective, dealing with a `string` is way easier than dealing with a slice (e.g. it can be `nil`). IngestChannel is a nice helper but not exactly everybody's first choice. You still get the TraceID from lower-level methods. Given that users probably want to hand it to use for investigating performance problems, unexpected behaviour, etc. they most likely need a reproducer first, that should ideally be a single request. IngestChannel still isn't a blackbox: It is instrumented via OpenTelemetry.

But Talking about OpenTelemetry: We could add the `axiom_trace_id` to every span? If people ever see something odd, they have the TraceID at hand?